easybuild/easyconfigs/l/libxc/libxc-3.0.0-intel-2017b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '3.0.0'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+# Results for some functionals (e.g. mgga_c_tpss) deviate with too aggressive optimization settings.
+# Tests also fail with Intel Compilers on Haswell when optarch is enabled.
+toolchainopts = {'lowopt': True, 'optarch': False}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/']
```

Comment:
@migueldiascosta please add checksums for new easyconfigs (pro tip: use `eb --inject-checksums`)